kite-data/kite-data-crunch/src/main/java/org/kitesdk/data/crunch/CrunchDatasets.java

```diff
@@ -96,6 +96,52 @@
   }
 
   /**
+   * Expose the given {@link View} as a Crunch {@link ReadableSource}.
+   *
+   * @param view the view to read from
+   * @param readerSchema the schema used to read from the view
+   * @param <E>     the type of entity produced by the source
+   * @return a {@link ReadableSource} for the view
+   *
+   * @since 1.1.0
+   */
+  public static <E> ReadableSource<E> asSource(View<E> view, Schema readerSchema) {
```

Comment:
I don't think we should pass both read schema and view. That allows setting the expected class and a projection, which I don't think should happen. Instead, I think the right thing to do is to extend views to handle column projection. They are half-way there by carrying the expected type already and the intent at the time (at least mine) was to finish the API by adding more variations for column projection.

Here are some methods we could add:
* `View<?>#as(Schema) => View<GenericRecord>`
* `View<?>#as(Class<T>) => View<T>`
* `View<?>#select(String...) => View<GenericRecord>`
* `View<?>#select(List<String>) => View<GenericRecord>`